src/testRunner/unittests/typingsInstaller.ts

```diff
@@ -954,36 +954,43 @@ namespace ts.projectSystem {
             const cachePath = "/a/cache";
             const node = {
                 path: cachePath + "/node_modules/@types/node/index.d.ts",
-                content: "export let x: number"
+                content: "export let x: number",
             };
-            const commander = {
-                path: cachePath + "/node_modules/@types/commander/index.d.ts",
-                content: "export let y: string"
+            const commanderJS: TestFSWithWatch.File = {

```

Comment:
I think you should add this as a separate test case (to ensure it keeps working when there is no resolution as one test case while when resolution is to js file as another test case)